test/JDBC/expected/sys-eomonth-vu-verify.out

```diff
@@ -0,0 +1,322 @@
+SELECT * FROM EOMONTH_EndOfMonthView ORDER BY DateValue ASC;
+GO
+~~START~~
+int#!#date#!#date
+1#!#1996-01-24#!#1996-01-31
+2#!#2000-06-15#!#2000-06-30
+3#!#2022-04-30#!#2022-04-30
+~~END~~
+
+
+EXEC GetEndOfMonthDate_EOMONTH 1;
+GO
+~~START~~
+date
+1996-01-31
+~~END~~
+
+
+SELECT * FROM EOMONTH_EndOfNextMonthView ORDER BY DateValue ASC;
+GO
+~~START~~
+int#!#date#!#date
+1#!#1996-01-24#!#1996-02-29
+2#!#2000-06-15#!#2000-07-31
+3#!#2022-04-30#!#2022-05-31
+~~END~~
+
+
+EXEC GetEndOfNextMonthDate_EOMONTH 1;
+GO
+~~START~~
+date
+1996-02-29
```

Comment:
I recommend adding a test for February in a leap year if that's not already included.

Reply:
Yes I have added the tests to 
--Check if its leap year. If it is a leap year and the month is feb it will return 29 otherwise 28.